scripts/exdump_monitor.sh

```diff
@@ -998,7 +974,7 @@ err8=`cat $DATA/error8`
 
 export STATUS=YES
 export DUMP_NUMBER=9
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 null
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 null
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.